src/test/add-comment.test.ts

```diff
@@ -0,0 +1,79 @@
+import { test, expect } from "@playwright/test";
+import express from "express";
+import { fileURLToPath } from "url";
+import path from "path";
+import { Server } from "http";
+
+const app = express();
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+const port = 8080;
```

Comment:
why is it hardcoded to 8080? what happens if i have something already running on that port?
will it run on 8081?

Reply:
solved!